t/json.t

```diff
@@ -26,6 +27,4 @@ is_deeply $db->query('select name from mojo_json_test where name like "%supergir
 is_deeply $db->query('select name from mojo_json_test where name like "%supergirl%"')->expand(1)->hash,
   {name => {nick => 'supergirl'}}, 'name as hash';
 
-$db->query('drop table mojo_json_test');
```

Comment:
I don't understand this change. Why don't you want to cleanup after the test is run? Also, why not use `if not exists` above?

Reply:
Why I don't want cleanup: Me personally I find it convenient to have tables containing some data after the tests.
Why no `if not exists`: If the tests fail no cleanup is made and the next time the tests start with a table already containing some data. Because of that the test `select id, name, j from mojo_json_test where json_extract(j, "$.foo") = 42` will not return `$$` as ID, but something else.

Reply:
I will allow this:

```
$db->query('drop table mojo_json_test') unless $ENV{TEST_KEEP_DB};
```

Reply:
Changed at 2 places and added 'truncate table' commands at the beginning.